### DISCUSSION TITLE
My Game is Broken /EXPLANATION/

### DISCUSSION THREAD

**Discussion by MemTheBun** [DEMOLITIONDON96/Demolition-Engine#81] [Lua Q&A]
So The mod in its beta the screen when i played the screen went down in game

Im Windows 11 player
     64x 
     8GB Ram

**Comment by ThatOneSillyGuyDon** [comment]
I’m a Windows 11 user, with the same specs as well (except ram, mine’s higher), and everything works fine, maybe try moving to the newest version of this engine under actions
